pdf/src/test/groovy/com/craigburke/document/builder/PdfDocumentLoader.groovy

```diff
@@ -68,16 +67,13 @@ class PdfDocumentLoader {
     }
 
     private static void loadChildren(Document document) {
-        def pages = document.element.documentCatalog.allPages
```

Comment:
signature of processStream has been changed and moreover this method has been private so we can not call it here.I was only left with processPage method but that also print the text of the processed page to "output". even this method can not be overridden in pdfextrator because anyways we end up calling super.processPage() for low level processing and it was throwing exception in write page call.
Suggest if there is another way of handling it.